### DISCUSSION TITLE
Urgent: Account Suspended Appeal - Missing Out on #100DaysOfOSS Challenge

### DISCUSSION THREAD

**Discussion by mayagriffith35** [community/community#136687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### GitHub Feature Area

GitHub Education

### Body

Hello GitHub Support Team,

My username is **https://github.com/madefromjames**, and my email is **madefromjames@gmail.com**. I’m reaching out to follow up on my account suspension, which happened six days ago. I’ve already submitted a ticket (Ticket Number: 2944993), but I haven’t received any updates yet.

My GitHub account was recently suspended while I was participating in the #100DaysOfOSS challenge, where I was sharing my progress and contributing to discussions within the open-source community.

I believe the suspension might have been triggered due to the inclusion of external links in my report, but I assure you that my intention was solely to share valuable content related to my open-source journey. I understand the importance of maintaining a safe and secure community, and I sincerely apologize if my actions inadvertently violated any guidelines.

I kindly request a review of my account suspension. I am more than willing to make any necessary adjustments to my post to comply with the community guidelines. This challenge is ongoing, and I’m eager to rejoin the community and continue contributing. I would greatly appreciate your prompt attention to this matter.

Thank you for your time and understanding. I look forward to your support in resolving this matter.

Best regards,
James Emmanuel

**Comment by ghostinhershell** [comment]
Hi @jamesenv,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by mayagriffith35** [reply]
@ghostinhershell I appreciate your time and effort with the feedback you provided. I'll patiently wait for it to be answered in the order it was received.

**Comment by FranPratama** [comment]
Take your time bud, I have submitted an appeal as well in 26th July 2024 but still no updates yet to this day 🙃
